src/setup/mod.rs

```diff
@@ -2,4 +2,6 @@
 
 pub mod config;
 pub mod node;
+pub mod process;
```

Comment:
not sure if we need an extra module just to start/stop the full node; also, the `process` name is easy to get mixed up with `std::process` or `tokio::process`

Reply:
Removed